HW5/28.py

```diff
@@ -0,0 +1,16 @@
+# Задача 28: Напишите рекурсивную функцию sum(a, b), 
+# возвращающую сумму двух целых неотрицательных чисел. 
+# Из всех арифметических операций допускаются только +1 и -1. Также нельзя использовать циклы.
+
+import random
+a = random.randint(0, 100)
+b = random.randint(0, 100)
+def sum(a, b):
+    if b == 0:
+        return a
+    elif b > 0:
+        return sum(a + 1, b - 1)
+    else:
+        return sum(a - 1, b + 1)
```

Comment:
На семинаре мы договорились, что на вход подаются неотрицательные числа, то есть проверка на положительность не нужна.
```suggestion

    return sum(a + 1, b - 1)
```